spec/services/courses/content_status_service_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+describe Courses::ContentStatusService do
+  let(:service) { described_class.new }
+  let(:execute_service) { service.execute(enrichment, next_cycle_boolean) }
+  let(:next_cycle_boolean) { false }
+
+  context "when the enrichment parameter is nil" do
+    let(:enrichment) { nil }
+
+    context "and belongs to the next recruitment" do
+      let(:next_cycle_boolean) { true }
```

Comment:
```suggestion
      let(:next_cycle_recruitment?) { true }
```

Any reason you didn't go with this?